portfolio/src/main/webapp/carousel.js

```diff
@@ -0,0 +1,16 @@
+// Automatic Slideshow - change image every 3 seconds
+var myIndex = 0;
+
+function carousel() {
```

Comment:
shall we name it startPlayingCarousel()? The current name `carousel()` doesn't seem to indicate what it is actually doing. But the file name is okay to be `carousel.js`

Reply:
ohh yeap that's true, I've changed that